README.md

```diff
@@ -2,17 +2,19 @@
 
 :link: [`coophub.io`](http://coophub.io)
 
-This web app uses the [GitHub API](https://developer.github.com/v3/) to join and show *nicely- the projects/repositories of any subscripted co-operative from over the world.
+This web app uses the [GitHub API](https://developer.github.com/v3/) and [GitLab API](https://docs.gitlab.com/ee/api/README.html) to join and show *nicely- the projects/repositories of any subscripted co-operative from over the world.
```

Comment:
Little fix for text: `...to fetch, process and nicely display the projects/repositories...`
The word "join" is confusing and `show nicely` is order-inverted.